sessionctx/variable/session.go

```diff
@@ -149,6 +149,9 @@ type SessionVars struct {
 	// GlobalAccessor is used to set and get global variables.
 	GlobalVarsAccessor GlobalVarAccessor
 
+	// LastFoundRows is the last found rows of previous query statement
```

Comment:
Maybe LastFoundRow is better than LastFoundRows. It is just a unit64 not a []uint64.